### PR TITLE
add browser tests for testing basic use of solidity tests tab

### PR DESCRIPTION
**Pull request by yann300**

### CODE REVIEW

File: test-browser/tests/solidityUnittests.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  },
+  tearDown: sauce
+}
+
+function runTests (browser) {
+  browser
+    .waitForElementVisible('#icon-panel', 10000)
+    .click('#icon-panel div[plugin="pluginManager"]')
+    .scrollAndClick('#pluginManager article[id="remixPluginManagerListItem_solidityUnitTesting"] button')
+    .clickLaunchIcon('fileExplorers')
```
Comment (LianaHus) [inline test-browser/tests/solidityUnittests.js]: shouldn't you activate solidity plugin and compile a file?

Comment (LianaHus) [reply test-browser/tests/solidityUnittests.js]: to run tests

Comment (yann300) [reply test-browser/tests/solidityUnittests.js]: it is already activated when tests loads. see init.js

Comment (yann300) [reply test-browser/tests/solidityUnittests.js]: and anyway remix-tests take care of compiling, so we only need the compile tab for getting the compiler version, not directly for compiling

**Pull request merged**